components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -102,6 +102,24 @@
  */
 @property(nonatomic, assign) CGFloat maximumInitialDrawerHeight;
 
+/**
+ A flag allowing clients to opt-out of the drawer closing when the user taps outside the content.
+
+ @default YES The drawer should autohide on tap.
+ */
+@property(nonatomic, assign) BOOL shouldAutoDismissOnTap;
+
+/**
+ A flag allowing clients to opt-in to handling touch events.
+
+ @default NO The drawer will not forward touch events.
```

Comment:
Please answer the following questions in the docs here:

- What is a touch event in this context?
- Where are the touch events being forward to?

Reply:
Yes, I’m not sure what the delegate means in this context. Is a specific selector invoked?

Reply:
Yes, I’m not sure what the delegate means in this context. Is a specific selector invoked?